runtime/doc/lsp.txt

```diff
@@ -71,12 +71,14 @@ options are not restored when the LSP client is stopped or detached.
 - |K| is mapped to |vim.lsp.buf.hover()| unless |'keywordprg'| is customized or
   a custom keymap for `K` exists.
 
-                                          *grr* *gra* *grn* *gri* *i_CTRL-S*
+                              *grr* *gra* *grn* *gri* *grs* *grS* *i_CTRL-S*
 Some keymaps are created unconditionally when Nvim starts:
 - "grn" is mapped in Normal mode to |vim.lsp.buf.rename()|
 - "gra" is mapped in Normal and Visual mode to |vim.lsp.buf.code_action()|
 - "grr" is mapped in Normal mode to |vim.lsp.buf.references()|
 - "gri" is mapped in Normal mode to |vim.lsp.buf.implementation()|
+- "grs" is mapped in Normal mode to |vim.lsp.buf.workspace_symbol()|
```

Comment:
would this be a good use of `gr/` or is that conceivably useful for something else (e.g. some sort of LSP-driven file navigation)?

Reply:
> would this be a good use of `gr/` or is that conceivably useful for something else (e.g. some sort of LSP-driven file navigation)?

I think "s" for symbol will make it easier to remember, "/" is associated with searching for something. `vim.lsp.buf.list_workspace_folders` seems like it has to do with some sort of navigation, (i haven't used it though). So maybe reserve `/` for that?

Reply:
>  "/" is associated with searching for something.

So we should reserve `gr/` for builtin (or delegated) "freceny"/fuzzy-finder (https://github.com/neovim/neovim/issues/660).

Reply:
Yes, solid idea